CHANGELOG.md

```diff
@@ -9,8 +9,17 @@ Please note that the date associated with a release is the date the code
 was committed to the `main` branch. This is not necessarily the date that
 the code was deployed.
 
+
+
```

Comment:
Can you please remove these extra lines for consistency with the rest of the file?